sdcm/utils/user_profile.py

```diff
@@ -117,12 +118,14 @@ def get_profile_content(stress_cmd):
 
     :return: (profile_filename, dict with yaml)
     """
-
-    cs_profile = re.search(r'profile=(.*\.yaml)', stress_cmd).group(1)
-    if "scylla-qa-internal" in cs_profile:
+    if "scylla-qa-internal" in stress_cmd:
+        cs_profile = stress_cmd
```

Comment:
this change is problematic, and not always correct, and gonna break @soyacz work

Reply:
I think we need here 2 regex of for `profile=...` and one for bare yaml,
and then apply logic regrading the name/origin of it.